### ISSUE TITLE
Bug query feature

### ISSUE THREAD

**Issue by justincred** [nus-cs2103-AY2324S2/forum#859]
Hi prof, if for example i implemented to check for overlap with everyone in schedule meeting but in reschedule meeting i only checked for overlap with itself, is it possible to fix the bug? to make reschedule check for overlap with everyone given that schedule does it

**Comment by damithc** [comment]
@justincred Yes, assuming the the overlap checking is already feature promised in the UG.
